### PR TITLE
Bug 1311760 - Add custom admin site for Cluster and SparkJob

### PR DESCRIPTION
**Pull request by maurodoglio**
This change is [Reviewable](https://reviewable.io/reviews/mozilla/telemetry-analysis-service/51)

### CODE REVIEW

File: atmo/clusters/admin.py [hunk 1]
```diff
@@ -5,6 +5,27 @@
...
+        'jobflow_id',
+        'emr_release',
+        'most_recent_status',
+    ]
+    list_filter = [
+        'most_recent_status',
+        'size',
+        'emr_release',
+    ]
+    search_fields = ['identifier', 'jobflow_id', 'create_by__email']
```
Comment (jezdez) [inline atmo/clusters/admin.py]: `s/create_by/created_by`

File: atmo/clusters/admin.py [hunk 2]
```diff
@@ -5,6 +5,27 @@
...
+    actions = [terminate, ]
```
Comment (jezdez) [inline atmo/clusters/admin.py]: mind removing the trailing comma?

File: atmo/jobs/admin.py [hunk 3]
```diff
@@ -3,6 +3,27 @@
...
+        'last_run_date',
+        'is_enabled',
+        'most_recent_status',
+    ]
+    list_filter = [
+        'most_recent_status',
+        'size',
+        'is_enabled',
+    ]
+    search_fields = ['identifier', 'is_enabled', 'create_by__email', 'most_recent_status']
```
Comment (jezdez) [inline atmo/jobs/admin.py]: `s/create_by/created_by`

Comment (jezdez) [inline atmo/jobs/admin.py]: Oh, not sure if `is_enabled` makes sense for `search_fields`. But maybe `current_run_jobflow_id`?

File: atmo/jobs/admin.py [hunk 4]
```diff
@@ -3,6 +3,27 @@
...
+        'identifier',
+        'size',
+        'created_by',
+        'start_date',
+        'end_date',
+        'last_run_date',
+        'is_enabled',
+        'most_recent_status',
+    ]
+    list_filter = [
```
Comment (jezdez) [inline atmo/jobs/admin.py]: Mind adding `start_date`, `end_date`, `last_run_date`, `interval_in_hours` to `list_filter`?

File: atmo/clusters/admin.py [hunk 5]
```diff
@@ -5,6 +5,27 @@
...
+        'size',
+        'created_by',
+        'start_date',
+        'end_date',
+        'jobflow_id',
+        'emr_release',
+        'most_recent_status',
+    ]
+    list_filter = [
+        'most_recent_status',
```
Comment (jezdez) [inline atmo/clusters/admin.py]: Mind adding the same fields like I mentioned for the job admin?

Comment (maurodoglio) [inline atmo/clusters/admin.py]: 👍

File: atmo/clusters/admin.py (hunk 1 above)
Comment (maurodoglio) [inline atmo/clusters/admin.py]: :+1:

File: atmo/clusters/admin.py (hunk 2 above)
Comment (maurodoglio) [inline atmo/clusters/admin.py]: 👍

File: atmo/jobs/admin.py (hunk 3 above)
Comment (maurodoglio) [inline atmo/jobs/admin.py]: 👍

**Pull request merged**